tests/python/gaia-ui-tests/gaiatest/apps/system/regions/cards_view.py

```diff
@@ -15,17 +15,19 @@ class CardsView(PageRegion):
 
     # Home/Cards view locators
     _root_locator = (By.ID, 'task-manager')
-    _cards_locator = (By.CSS_SELECTOR, '#cards-view .card')
+    _card_locator = (By.CSS_SELECTOR, '#cards-view .card')
+    _cards_no_recent_windows_locator = (By.ID, 'cards-no-recent-windows')
+    _new_private_sheet_locator = (By.ID, 'task-manager-new-private-sheet-button')
+    _new_sheet_locator = (By.ID, 'task-manager-new-sheet-button')
+
     # Check that the origin contains the current app name, origin is in the format:
     # app://clock.gaiamobile.org
+    # This locator is used by is_app_a11y_visible
     _apps_cards_locator = (By.CSS_SELECTOR, '#cards-view li[data-origin*="%s"]')
-    _close_buttons_locator = (By.CSS_SELECTOR, '#cards-view li[data-origin*="%s"] .close-button')
-
-    _new_private_sheet_locator = (By.ID, 'task-manager-new-private-sheet-button')
-    _new_sheet_locator = (By.ID, 'task-manager-new-sheet-button')
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
```

Comment:
CardsView is a PageRegion. It'd better if we call the PageRegion's contructor here.